client/ayon_core/hosts/houdini/plugins/create/create_hda.py

```diff
@@ -16,6 +17,9 @@ class CreateHDA(plugin.HoudiniCreator):
     icon = "gears"
     maintain_selection = False
 
+    min_num_inputs = 0
+    max_num_inputs = 0
```

Comment:
I feel like these class attributes are not needed. See [here](https://github.com/ynput/ayon-core/pull/456/files#r1584444651).

I'd recommend removing these and just setting the defaults directly in the `EnumDef` since this doesn't seems like something that needs to be exposed in settings?

Also, is `0` the good "default"? Wouldn't it usually default to `1`?

Reply:
`0` is always the default for `min_num_inputs` and `max_num_inputs` in `hou.Node.createDigitalAsset(...)` 

```python
createDigitalAsset(self, name=None, hda_file_name=None,
    description=None, min_num_inputs=0, max_num_inputs=0,
    compress_contents=False, comment=None, version=None,
    save_as_embedded=False, ignore_external_references=False,
    change_node_type=True, create_backup=True, install_path=None) -> Node
```

Reply:
I agree that this should be removed unless we have these attributes in the houdini addon setting.
The one from the enum definition should be enought for now.

Reply:
I also agree, but how can I pass the `pre_create_data` to `create_instance_node` ? 
`create_instance_node` is called inside a super method that doesn't pass that argument.

This mentioned here https://github.com/ynput/ayon-core/pull/456#discussion_r1586038828


what about saving `pre_create_data` to `self.pre_create_data` 
and  use it inside `create_instance_node` ?

Reply:
Yes, it would be along the lines of https://github.com/BigRoy/ayon-core/commit/6e68802c7617e1aba635be89d60051583e1226ff

Reply:
oh, thanks!

Reply:
Are we still keeping these two variables or we can just set the values in the number definitions?

Reply:
They are used inside `create_instance_node` as a fall back value.

https://github.com/ynput/ayon-core/blob/79139ce0dd4b31c32c98d26403a2f78c6e345f14/client/ayon_core/hosts/houdini/plugins/create/create_hda.py#L51-L54